nbgrader/apps/baseapp.py

```diff
@@ -259,13 +263,13 @@ def initialize(self, argv=None):
     def init_src(self):
         """Compute and check the source paths for the transfer."""
         raise NotImplemented
-    
+
```

Comment:
I can't comment on the relevant line, but I think the `init_args` in this class can also be removed and the call to it in the `initialize` function.